### ISSUE TITLE
Add 404 Page

### ISSUE THREAD

**Issue by harshkhandeparkar** [kossiitkgp/KWoC-21-Frontend#192]
Self explanatory.

**Comment by visheshrwl** [comment]
@harshkhandeparkar please assign me this

**Comment by VineeTagarwaL-code** [comment]
i would like to work on this issue

**Comment by snowden-fu** [comment]
@harshkhandeparkar plz review my solution

**Comment by harshkhandeparkar** [comment]
@fuzheng1998 Please do not open a PR before being assigned to the issue. Multiple people may end up working on the same issue :)

**Comment by harshkhandeparkar** [comment]
> @harshkhandeparkar please assign me this

@visheshrwl Sorry, I missed this yesterday. Since @fuzheng1998 has already opened a PR for this, would you be interested in working on another issue instead?
